### DISCUSSION TITLE
DynDNS Client integration possible?

### DISCUSSION THREAD

**Discussion by jojo-gfx** [tobychui/zoraxy#365] [Ideas]
All time I have the problem to manage a DynDNS client anywhere in my network. It is a service, who essentially is need for a reverse proxy to work and so it would be perfect, the reverse proxy manager can also manage the DynDNS configuration.
At time I was go back to the service of my fritzbvox, but it can only handle one provider and at time I have a second provider and have now again the problem, where could I run the DynDNS client?

I have an eye on zoraxy but not installed yet, so sorry, if it is already installed (I've not seen it in docs). ;)
If that feature would be integrated, so zoraxy would set the level up. :)

Greetings

**Comment by tobychui** [comment]
I don't know what a DynDNS client is, but it do feels like a Dynamic DNS client. If it is really the case, you supposing do not need a reverse proxy in front of it to make it work. It will just works by itself running in another process and update the DNS record for you via your DNS service / domain name provider.

**Reply by jojo-gfx** [reply]
Yes it is what I mean. 
Yes, it is a small software but also with absolutely no comfort at time. I know it only per shell and config files and so it's hidden and fast forget, where it is running.
I think the most use case for a reverse proxy is to split incoming network traffic to servers from dynamic dns domains so I would see the perfect place for the management also in the reverse proxy manager.
Would think nobody has a static IP at home... 🤔
Greetings

**Reply by tobychui** [reply]
@jojo-gfx Hmm, I guess you are not using your DDNS correctly. According to my knowledge about DDNS and how DDNS clients usually work, a DDNS client do not need to be reachable from the internet. As soon as the client can access internet from your host (e.g. under an NAT environement with no port forward to your DDNS client), it will work.

You should be using CNAME records for your subdomains and a single A record that points to your own IP. Although loop-back setups (subdomain to your reverse proxy and use proxy routing to your primary domain) will also works but that will halve the incoming traffic bandwidth.

**Reply by jojo-gfx** [reply]
Don't know what you mean?
My Dynymic DNS Client is at time my Fritzbox, but they can only contact one hoster/provider and so far it is ok.
But now I, and many others, have to set the correct IP to several providers of domains and so it is neccessary to use a software like ddclient and run it on a machine anywhere in the network. But thats no good idea, because you have to think on it and also the configuration isn't comfortable.
Because the ddclient is only running for my dynamic dns states (update the IPs) it is directly important for the reverse proxy also, because no correct IP in Dynamic DNS Settings of providers lead to no function of all my domains/subdomains and so the proxy is useless.

Sorry, I'm not a native english writer so I only can hope you understand what I mean: It is one thing, the reverseproxy needs a correct running dynamic DNS client.

;)

**Comment by BrightRaider** [comment]
QuickSave is now live in V9.6! It covers exactly what you described — one keypress moves the Snap Hook between any inventory slot and your Safe Pocket (or back). 5 independent presets, toggle direction per preset, all timing configurable. The Snap Hook is safe.

Release coming soon.
